DigitalLearningSolutions.Web/Views/Shared/SearchablePage/_SearchResultsCount.cshtml

```diff
@@ -5,5 +5,5 @@
   @{
     var count = Model.MatchingSearchResults;
   }
-  <span role="alert" id="results-count">@count matching @(count == 1 ? "result" : "results")</span>
+  <span id="results-count">@count matching @(count == 1 ? "result" : "results")</span>

```

Comment:
Not sure this is the best idea. This will presumably get rid of all announcements of the number of results. If we are fine with that, then cool. Otherwise, there is an alternative method. 

The SearchSortFilterAndPaginate js file updates the alert count, but if the count is the same as it was before, we add some whitespace to it to force screen readers to announce it again. I think I added this to prevent confusion when typing quickly, I can look into that reasoning more if it doesn't make sense. 

Previously, we had a parameter somewhere in the SearchSortFilterPaginate file to not update the result count in JS when searching/sorting/filtering/paginating. This could then be true in the constructor, to avoid "updating" the result count on page load. I'm not sure if that was removed, or the reasoning why, might be able to look through the changes to see. But reinstating that might work, while keeping the regular announcements.

Reply:
Alex and I were talking about it and we couldn't think of much benefit in having this read out as an alert. The page number is also an alert field, so if the number of pages change (or the number of the page the user is on changes) then that will get read out. I now realise that if filtering reduces the number of pages to 1 the page element disappears and doesn't get read out.

Revolutionary idea though, what if we were to combine the searchResultsCount alert and the page number alert in a hidden span? Then we have only one alert and get both when either number changes.

Reply:
That might work, will give it a try

Reply:
I think the main benefit of the result count alert was so users would get updated that something has "happened" when they are typing in a search box for instance. A user that doesn't need a screen reader will easily notice that the page number and result count have changed, and an alert would help a screen reader user be similarly aware of it. 

Hidden span might work, would have to be careful with how we hide it to make sure the screen reader still announces it. Though it does make more sense to me to use the already visually displayed page elements as alerts over introducing new hidden ones.

Reply:
The motivation for removing one of them was that they seemed to be interrupting each other on Firefox so that only one was getting read out. So 1 should be better than 2.